lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -269,6 +270,40 @@ export class NativeInteractionClient extends BaseInteractionClient {
         }
     }
 
+    /**
+     * Gets MATS telemetry from native response
+     * @param response 
+     * @returns 
+     */
+    private getMATSFromResponse(response: NativeResponse): object|null {
+        let mats = null;
+        if (response.properties.hasOwnProperty(NativeConstants.MATS_TELEMETRY)) {
+            try {
+                mats = JSON.parse(response.properties[NativeConstants.MATS_TELEMETRY]);
```

Comment:
Would be fine with immediately returning so that you don't need to instantiate `mats` outside of the try.

Reply:
Done